src/api/app/models/local_build_result/for_package.rb

```diff
@@ -0,0 +1,45 @@
+class LocalBuildResult
+  class ForPackage
+    include ActiveModel::Model
+    attr_accessor :package, :project, :show_all
+    attr_reader :excluded_counter, :results
+
+    def initialize(attributes = {})
+      super
+      @excluded_counter = 0
+      buildresults
+    end
+
+    private
+
+    attr_writer :excluded_results_count, :results
+
+    def buildresults
+      @results = {}
```

Comment:
You could do this in the getter, of course you don't need the attr_reader anymore:

```
def results
  @results || {}
end
```